js.test

```diff
@@ -0,0 +1,14 @@
+var movies = ["The Shawshank Redemption",
+"The Godfather",
+"The Dark Knight",
+"12 Angry Men",
+"The Lord Of The Rings",
+"Pulp Fiction",
+"The Good, the Bad and the Ugly",
+"Forrest Gump"];
+
+console.log(movies);
```

Comment:
This does display the titles of the movies, but you did not use a `for` loop, as prescribed. You are just writing out the array as a whole. Again, even if you did not know how to do this from memory, just use the exercise text, and write `javascript loop through array with for` into Google, which gave me [this](https://www.freecodecamp.org/news/how-to-loop-through-an-array-in-javascript-js-iterate-tutorial/) as the second page, containing the solution. Stay curious!